src/applications/mhv-secure-messaging/components/ComposeForm/RecipientsSelect.jsx

```diff
@@ -78,13 +79,26 @@ const RecipientsSelect = ({
 
   const handleRecipientSelect = useCallback(
     e => {
-      const recipient = recipientsList.find(r => +r.id === +e.detail.value);
-      setSelectedRecipient(recipient);
-      if (recipient.signatureRequired || isSignatureRequired) {
+      const recipient = recipientsList?.find(r => +r.id === +e.detail.value);
+
+      if (e.detail.value === '') {
+        setSelectedRecipient({});
```

Comment:
Just curious why do we need to set this to a blank object `{}` if normal values are strings? should it be `null` or `' '` ?

Reply:
Having `null` or an `' '` here caused a small bug that would not focus on the select field when the value was deselected. We need the blank `{}` here because `selectedRecipient` and `recipient` are returning objects when selected. If `e.detail.value` has an empty value, `' '`, then the `selectedRecipient` should be an empty `{}`.

Reply:
understood. As as we tested during code pairing, we did ensure that selectedRecipient is populated with an object, so that makes sense